### PR TITLE
Minivans are not cool - Toyota Sienna Port

### PR DESCRIPTION
**Pull request by wocsor**
I'll grab a cabana drive with OP, but @russellpstrom on Discord has said that it "drives great." Simple enough to port, requires switch config 0101 and DSU unplug for full long control.

### CODE REVIEW

**Comment by rbiasini** [comment]
great, the long awaited Sienna :) PR looks good. I'll hold on for the cabana drive.

**Comment by wocsor** [comment]
ok here's a link, unfortunately it's not the "copy share link" but hopefully you guys can access the drive 
https://my.comma.ai/cabana/?route=eb6acd681135480d%7C2019-06-20--20-00-00&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2Feb6acd681135480d%2F8e5e45f6aab3c30e6ae8a60bd6e2aefb_2019-06-20--20-00-00&seekTime=596

**Comment by rbiasini** [comment]
@wocsor , could you please rebase it on latest devel? Otherwise it seems ready to go.
Thanks!

**Comment by wocsor** [comment]
10-4, will try to get to this tonight!

**Comment by wocsor** [comment]
Rebased on 0.6 devel

**Comment by geohot** [comment]
@rbiasini Any objections to me merging this?

**Comment by rbiasini** [comment]
merging internally first. Will be in 0.6.1.

**Comment by rbiasini** [comment]
@wocsor this is the fingerprint I get from your drive:
```
36: 8, 37: 8, 114: 5, 119: 6, 120: 4, 170: 8, 180: 8, 186: 4, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 544: 4, 545: 5, 548: 8, 550: 8, 552: 4, 562: 4, 608: 8, 610: 5, 643: 7, 705: 8, 725: 2, 740: 5, 764: 8, 800: 8, 824: 8, 835: 8, 836: 8, 849: 4, 869: 7, 870: 7, 871: 2, 888: 8, 896: 8, 900: 6, 902: 6, 905: 8, 911: 8, 916: 1, 918: 7, 921: 8, 933: 8, 944: 6, 945: 8, 951: 8, 955: 8, 956: 8, 979: 2, 992: 8, 998: 5, 999: 7, 1000: 8, 1001: 8, 1002: 8, 1008: 2, 1014: 8, 1017: 8, 1041: 8, 1042: 8, 1043: 8, 1056: 8, 1059: 1, 1076: 8, 1077: 8, 1114: 8, 1160: 8, 1161: 8, 1162: 8, 1163: 8, 1164: 8, 1165: 8, 1166: 8, 1167: 8, 1176: 8, 1177: 8, 1178: 8, 1179: 8, 1180: 8, 1181: 8, 1182: 8, 1183: 8, 1191: 8, 1192: 8, 1196: 8, 1197: 8, 1198: 8, 1199: 8, 1200: 8, 1201: 8, 1202: 8, 1203: 8, 1212: 8, 1227: 8, 1228: 8, 1235: 8, 1237: 8, 1279: 8, 1552: 8, 1553: 8, 1555: 8, 1556: 8, 1557: 8, 1561: 8, 1562: 8, 1568: 8, 1569: 8, 1570: 8, 1571: 8, 1572: 8, 1584: 8, 1589: 8, 1592: 8, 1593: 8, 1595: 8, 1656: 8, 1664: 8, 1666: 8, 1667: 8, 1728: 8, 1745: 8, 1779: 8, 1904: 8, 1912: 8, 1990: 8, 1998: 8
```
Not seeing msg 238

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -198,8 +202,9 @@ def check_ecu_msgs(fingerprint, ecu):
   CAR.RAV4_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
   CAR.COROLLA_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
   CAR.LEXUS_ESH_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
+  CAR.SIENNA: dbc_dict('toyota_sienna_xle_2018_pt_generated', 'toyota_adas'),
 }
 
 NO_DSU_CAR = [CAR.CHR, CAR.CHRH, CAR.CAMRY, CAR.CAMRYH, CAR.RAV4_TSS2, CAR.COROLLA_TSS2, CAR.LEXUS_ESH_TSS2]
 TSS2_CAR = [CAR.RAV4_TSS2, CAR.COROLLA_TSS2, CAR.LEXUS_ESH_TSS2]
-NO_STOP_TIMER_CAR = [CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.RAV4_TSS2, CAR.COROLLA_TSS2, CAR.LEXUS_ESH_TSS2]  # no resume button press required
+NO_STOP_TIMER_CAR = [CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.RAV4_TSS2, CAR.COROLLA_TSS2, CAR.LEXUS_ESH_TSS2, CAR.SIENNA]  # no resume button press required
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: oh, it can do stop and go with no button press?

Comment (wocsor) [reply selfdrive/car/toyota/values.py]: according to the owner. i was surprised, it has the same resume bug as the RAV4 hybrid

**Comment by wocsor** [comment]
sorry, I had used a fingerprint from the original fork instead of collecting one from scratch. since there was only one tester left I can't confirm that the fingerprint came from him

**Comment by rbiasini** [comment]
ok, I'll use what I can see from the route you posted.

**Comment by rbiasini** [comment]
merged internally. Will be in next release, thanks.

**Pull request closed without merging**